tests/boost/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+cmake_minimum_required(VERSION 3.18)
```

Comment:
https://cmake.org/cmake/help/latest/command/file.html#archive-extract

The upstream CMake scripts do no install the headers of config files so I opted to keep the "install" here instead of making an action for it